tests/rtp-mode.rs

```diff
@@ -0,0 +1,140 @@
+use std::collections::VecDeque;
+use std::net::Ipv4Addr;
+use std::time::Duration;
+use tracing::info_span;
+
+use str0m::format::Codec;
+use str0m::media::rtp::RtpHeader;
+use str0m::media::rtp::{Extension, ExtensionMap};
+use str0m::media::{Direction, MediaKind};
+use str0m::{Candidate, Event, Rtc, RtcError};
+
+mod common;
+use common::{init_log, progress, TestRtc};
+
+#[test]
+pub fn rtp_mode() -> Result<(), RtcError> {
+    init_log();
+
+    let rtc1 = Rtc::builder().set_rtp_mode(true).build();
+    let rtc2 = Rtc::builder()
+        .set_rtp_mode(true)
+        // release packet straight away
+        .set_reordering_size_audio(0)
```

Comment:
if this is set to 4 will the test still pass?

Reply:
when packet is lost

Reply:
I think it will release the first two in sequence and not the last one.

Reply:
I see I need more packets to get it to release the last one.

Reply:
yes. it needs to "push out"